Analysis/ToothFractureStatistics/ToothFractureStatistics/Testing/Python/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+
+#slicer_add_python_unittest(SCRIPT ${MODULE_NAME}ModuleTest.py)
```

Comment:
Ah that was from the slicer extension template. I didn't change it. I'll add a comment.